tests/integration/tier0/non-destructive/subscription-manager/test_sub_man_pre_register.py

```diff
@@ -75,3 +75,43 @@ def test_unregistered_no_credentials(shell, convert2rhel):
         c2r.expect("SUBSCRIBE_SYSTEM::SYSTEM_NOT_REGISTERED - Not registered with RHSM", timeout=300)
 
     assert c2r.exitstatus != 0
+
+
+@pytest.mark.test_no_sca_not_subscribed
+def test_no_sca_no_subscribed(shell, pre_registered, convert2rhel):
+    """
+    This test verifies that running conversion on pre-registered system
+    without an attached subscription will try auto attaching the subscription.
+    """
+    with convert2rhel("--debug") as c2r:
+        # We need to get past the data collection acknowledgement.
+        c2r.expect("Continue with the system conversion?")
+        c2r.sendline("y")
+
+        c2r.expect("We'll try to auto-attach a subscription")
```

Comment:
Is there some additional message saying something that the auto-attachment was successful?

Reply:
Done @kokesak

Reply:
Danke!